### PR TITLE
Amazon KMS + SES extensions

### PR DESCRIPTION
**Pull request by marcinczeczko**
Amazon KMS (key management service) + SES (Simple email service) extensions

### CODE REVIEW

**Comment by hrstoyanov** [comment]
Hopefully, this makes it in 1.5!

**Review by gsmet — CHANGES_REQUESTED** [review]
Looks good to me, thanks! Do you think you will be able to write the guides before next Tuesday? Just to know if we should target 1.5 or push this to 1.6.

**Comment by marcinczeczko** [comment]
I might not have it ready by Tuesday. So 1.6 seems more realistic

śr., 20 maj 2020, 10:01 użytkownik Guillaume Smet <notifications@github.com>
napisał:

File: .github/workflows/ci-actions.yml [hunk 1]
```diff
@@ -486,7 +488,7 @@ jobs:
         if: matrix.mssql
       - name: Amazon Services
         run: |
-          docker run --rm --publish 8000:4569 --publish 8008:4572 --publish 8009:4575 --publish 8010:4576 --name build-amazon-service-clients -e SERVICES=s3,dynamodb,sns,sqs -e START_WEB=0 \
+          docker run --rm --publish 8000:4569 --publish 8008:4572 --publish 8009:4575 --publish 8010:4576 --publish 8011:4599 --publish 8012:4566 --name build-amazon-service-clients -e SERVICES=s3,dynamodb,sns,sqs,kms,ses -e START_WEB=0 \
```
Comment (gsmet) [inline .github/workflows/ci-actions.yml:491]: You probably need to make the timeout for the Amazon services native job higher. It's 20 right now, maybe make it 30.

Comment (marcinczeczko) [reply .github/workflows/ci-actions.yml:491]: Yep, i noticed that

**Comment by marcinczeczko** [comment]
@gsmet - I managed to produce guides and quickstart.

**Comment by gsmet** [comment]
OK, I will merge that one and backport. I will probably tweak the guides a bit later if I have some time.

Thanks for the quick turnaround!

**Review by gsmet — APPROVED** [review]

**Pull request merged**